components/molecule/phoneValidation/src/index.js

```diff
@@ -0,0 +1,145 @@
+import {useEffect, useRef, useState} from 'react'
+
+import cx from 'classnames'
```

Comment:
🚨 `classnames` is not included in the component dependencies. Add it with other components versions
`npm list classnames`

Reply:
Thanks! Didn't notice about it.